src/libcore/num/mod.rs

```diff
@@ -4449,6 +4449,9 @@ macro_rules! try_from_unbounded {
         impl TryFrom<$source> for $target {
             type Error = TryFromIntError;
 
+            /// Try to create the target type from the source type.
+            /// This particular variant will never fail, but is included
+            /// for completeness's sake.
```

Comment:
For some `TryFrom` conversions that involve `usize` or `isize`, details of how the conversion is implemented depend on the target platform (’s pointer size). But we typically only generate one set of documentation for the standard library, so I think we shouldn’t document details that only apply to some platforms.

The easiest way would be to have the same doc-comment for all `impl` blocks that use `TryFromIntError`.